websites/L/Lazada/metadata.json

```
@@ -10,11 +10,17 @@
 	},
 	"url": [
 		"lazada.com",
-		"www.lazada.com"
+		"www.lazada.com",
+		"lazada.co",
+		"www.lazada.co",
+		"lazada.vn",
+		"www.lazada.vn",
+		"lazada.sg",
+		"www.lazada.sg"
 	],
 	"version": "1.0.1",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png",
-	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/thumbnail.jpg",
+	"logo": "https://i.ibb.co/LpxnMfj/512x512pngoflazada.png",
+	"thumbnail": "https://www.most2414.com/wp-content/uploads/2020/02/02FEB-Insights-Lazada-Service-Marketplace-MOST2414-Cover.jpg",
```

These assets appear to be the same
```suggestion
	"logo": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png",
	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/thumbnail.jpg",
```